### ISSUE TITLE
Negative integers are not parsed

### ISSUE THREAD

**Issue by LeoniePhiline** [DerekStride/tree-sitter-sql#103]
Issue is `_number` being used as `choice` in `literal`, but also as part of `_decimal_number`.
```
    literal: $ => prec(2,
      choice(
        $._number,
        $._decimal_number,
        $._literal_string,
        $.keyword_true,
        $.keyword_false,
        $.keyword_null,
      ),
    ),
    _double_quote_string: _ => seq('"', /[^"]*/, '"'),
    _literal_string: $ => choice(
      seq("'", /[^']*/, "'"),
      $._double_quote_string,
    ),
    _number: _ => /\d+/,
    _decimal_number: $ => choice(
        seq(optional("-"), ".", $._number),
        seq(optional("-"), $._number, ".", $._number),
        seq(optional("-"), $._number, "."),
    ),
```

Therefore, `\d+` is correct, and a new anonymous option `_integral_number` must be added, using `seq(optional("-"), $._number)`.

**Comment by LeoniePhiline** [comment]
Fixed by https://github.com/DerekStride/tree-sitter-sql/pull/102/commits/756ffc20c19c2cfb01c52c26938fe10b43d9fd4d as part of #102.

**Comment by DerekStride** [comment]
Closed in #102
